opm/parser/eclipse/Parser/MessageContainer.hpp

```diff
@@ -43,21 +41,31 @@ namespace Opm {
 
 
     struct Location {
+        Location() = default;
+        Location( const std::string& s, int l ) : filename( s ), lineno( l ) {}
```

Comment:
Actually a slight oversight on my part, this was mostly thrown in for testing the emplace_back syntax in https://github.com/OPM/opm-parser/pull/748#discussion_r58354804

I